### DISCUSSION TITLE
Using docker, can't login

### DISCUSSION THREAD

**Discussion by ayman999uasy** [hoppscotch/hoppscotch#2942] [Q&A]
Running docker hoppscotch
http://45.33.68.32:3000/

Can not login with any service (google, github, microsoft) the error message (somwthing went wrong)

also can not log with email, error message: firebase: Error (auth/requests-from-referer-http://45.33.68.32:3000-are-blocked.)

**Comment by sschneider-ihre-pvs** [comment]
same here

**Comment by eduardoitelisoft** [comment]
You need to create a firebase project and init the docker environment variables.

https://github.com/hoppscotch/hoppscotch/blob/main/.env.example

**Comment by dancabper** [comment]
Hi, I have put the microsoft credentials and when I login it returns the error "AADSTS900971: No reply address provided". ¿You know what is the correct reply address url?
